### ISSUE TITLE
Adding SWC support for faster compiling time. 

### ISSUE THREAD

**Issue by richardscull** [NatsuYamaoka/YamaokaDiscord#17]
I got fed up with long compiling times while testing stuff, so after digging around, I found that adding `--swc` flag for ts-node actually gives us 2x boost in compiling time. 

Created a branch with a testing file, so you could give it a try yourself. Just don't forget to `npm i` new depends. Testing file in main directory, called `test.bat`. 

My results:
```bash
> ./test.bat
Calling without --swc flag
Without --swc flag: 4021 milliseconds
Calling with --swc flag
With --swc flag: 2014 milliseconds
```
Branch itself: https://github.com/shysecre/yamaoka-discord/tree/feature/swc-usage 

-----------

💡 On a side note, look for another ideas to optimize compiling time for testing environment.

**Comment by richardscull** [comment]
@shysecre ⬆️

**Comment by richardscull** [comment]
Resolved by #18
